requirements.in

```diff
@@ -2,10 +2,9 @@ dune-client
 moralis
 pandas
 pandas-stubs
-psycopg2
+psycopg
```

Comment:
Can you elaborate a bit on this change? E.g., psycopg2 is what we use in other repos as well.

Reply:
Probably not strictly required anymore. I used some features only in the new version (psycopg) but the current code would also work with the old version (psycopg2).

I would generally say that we should use current stable version. psycopg (which is the new version, sometimes called psycopg 3) has been [out since 2021](https://pypi.org/project/psycopg/#history). The old version pycopg2 has [only received bug fixes since then](https://pypi.org/project/psycopg2/#history).

Reply:
Ok good to know. Yeah i don't mind, it was mainly out of curiosity as i wasn't aware when each one was released.